### PR TITLE
client side support for realm private info endpoint

### PR DESCRIPTION
**Pull request by rosasck**
The client side support for the new identity endpoint for fetching private realm info

### CODE REVIEW

File: identityClient/api.go [hunk 1]
```diff
@@ -60,6 +60,14 @@ type RealmInfo struct {
 	Domain                string    `json:"domain"`
 }
 
+// PrivateRealmInfo represents the private information about a TozID realm
+type PrivateRealmInfo struct {
```
Comment (galxy25) [inline identityClient/api.go:64]: `PrivateRealmInfoRequest`

Comment (galxy25) [reply identityClient/api.go:64]: never mind...

Comment (galxy25) [reply identityClient/api.go:64]: but you need to figure out what to do with name, either the method needs to fill it in or pull it from the response struct

**Review by galxy25 — APPROVED** [review]

File: identityClient/identityClient.go [hunk 2]
```diff
@@ -676,6 +676,18 @@ func (c *E3dbIdentityClient) RealmInfo(ctx context.Context, realmName string) (*
 	return info, err
 }
 
+// PrivateRealmInfo fetches the private realm information based on realm name.
+func (c *E3dbIdentityClient) PrivateRealmInfo(ctx context.Context, realmName string) (*PrivateRealmInfo, error) {
+	var info *PrivateRealmInfo
+	path := c.Host + identityServiceBasePath + "/realm/info" + strings.ToLower(realmName)
```
Comment (galxy25) [inline identityClient/identityClient.go:682]: The endpoint should handle lower casing the realm name, having `strings.ToLower` is an anti pattern / code smell

Comment (galxy25) [reply identityClient/identityClient.go:682]: (updated)

**Review by galxy25 — APPROVED** [review]

**Review by efabens — APPROVED** [review]

**Review by lkwdwrd — APPROVED** [review]

**Comment by rosasck** [comment]
copy pasta tests

**Pull request merged**